raiden/network/rpc/client.py

```diff
@@ -361,6 +361,9 @@ def manager_by_token(self, token_address):
         if token_address not in self.token_to_channelmanager:
             token = self.token(token_address)  # check that the token exists
             manager_address = self.default_registry.manager_address_by_token(token.address)
+            if manager_address == '':
+                raise ValueError("Manager for token 0x%s does not exist" %
```

Comment:
Please use single quotes and `.format`  to keep the code consistent with the rest of the code base